internal/turso/databases.go

```diff
@@ -199,6 +201,34 @@ func (d *DatabasesClient) Update(database string) error {
 	return nil
 }
 
+type Body struct {
+	Org string `json:"org"`
+}
+
+func (d *DatabasesClient) Transfer(database, org string) error {
+	url := d.URL(fmt.Sprintf("/%s/transfer", database))
+	body, err := json.Marshal(Body{Org: org})
+	bodyReader := bytes.NewReader(body)
+	if err != nil {
+		return fmt.Errorf("could not serialize request body: %w", err)
+	}
+	r, err := d.client.Post(url, bodyReader)
+	if err != nil {
+		return fmt.Errorf("failed to transfer database")
+	}
+	defer r.Body.Close()
+
+	if r.StatusCode == http.StatusForbidden {
+		return fmt.Errorf("user is not an owner of the organizations involved in the transfer")
+	}
```

Comment:
This error message could be more useful if it said which organization the user was not an owner of (assuming that the requirement is that the user is the owner of *both* of the orgs).

Reply:
Yes! Now it is better
<img width="814" alt="Screen Shot 2023-07-06 at 4 04 46 PM" src="https://github.com/chiselstrike/turso-cli/assets/11340665/f1f98d49-1570-4fda-8e28-dc7a8837bfd6">

Reply:
It is using the response of the API to provide this error msg instead of building it in the command line application